runtime-sofa-boot-starter/src/main/java/com/alipay/sofa/runtime/annotation/PlaceHolderBinder.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.runtime.annotation;
+
+/**
+ * @author qilong.zql
+ * @since 2.5.2
+ */
+public interface PlaceHolderBinder {
+    String bind(String string);
```

Comment:
Please add comment for this interface and method. And should we provide a default Implement?  I see a lot of duplicate implements below.